### DISCUSSION TITLE
clone vs dup?

### DISCUSSION THREAD

**Discussion by emca9711** [TriliumNext/Trilium#1997] [Q&A]
I just want to clarify something. If I want an *independent* copy of a note - say I want to keep the demo files unchanged but want to copy some of them and edit only the copies, I can duplicate them rather than cloning them, correct? If so, this may be a good thing to add into the wiki :-)

Thanks for a great product, by the way. I'm very much enjoying it!

**Comment by zadam** [comment]
Yes, that's correct. Duplication will produce a complete independent copy.
